src/main.ts

```diff
@@ -163,7 +163,7 @@ export async function run(): Promise<void> {
       .trim()
       .split('\n')
       .map(line => line.trim())
-    const repoName = `${forkedModRepo.owner.login}/${forkedModRepo.name}`
+    const repoName = `${github.context.repo.owner}/${github.context.repo.repo}`
```

Comment:
Why was this changed?

Reply:
It was wrong.  I wanted the context, not the target repo.

repo name is used for the blacklist